src/Underworld.ts

```diff
@@ -394,22 +394,24 @@ export default class Underworld {
       // and is "complete"
       return true;
     }
-    const trueVelocity = Vec.multiply(deltaTime, velocity);
-    const trueMagnitude = Vec.magnitude(trueVelocity);
-    if (trueMagnitude <= 0.1) {
+    // Represents the positional offset for this simulation loop
+    // AKA the Vec2 distance to travel
+    const deltaPosition = Vec.multiply(deltaTime, velocity)
+    if (Vec.sqrMagnitude(deltaPosition) < 0.01) {
```

Comment:
Why is sqrMagnitude used here instead of magnitude?

Reply:
Just a little optimization. It has no other differences or impact on the game 👍 See my other comment for more info